server/chat/backend/agent/orchestrator/findings_schema.py

```
@@ -0,0 +1,102 @@
+            f"findings.md status must be one of {sorted(_VALID_STATUSES)}, got {status!r}"
+        )
+    strength = str(meta.get("self_assessed_strength", ""))
+    if strength not in _VALID_STRENGTHS:
+        raise FindingsValidationError(
+            f"findings.md self_assessed_strength must be one of {sorted(_VALID_STRENGTHS)}, got {strength!r}"
+        )
+    for section in _REQUIRED_SECTIONS:
+        if section not in content:
+            raise FindingsValidationError(
+                f"findings.md missing required section: {section!r}"
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)